src/cr/cube/matrix.py

```diff
@@ -279,7 +279,16 @@ def zscores(self):
         deviations above (positive) or below (negative) the population mean a cell's
         value is.
         """
-        raise NotImplementedError
+        # --- punt except on CAT_X_CAT (if any MR dimensions it gets np.nans) ---
```

Comment:
What happens with `CA_SUBVAR`?

Reply:
We don't distinguish CA_SUBVAR from CAT for matrix purposes. Do we need to in this case? If so, it wasn't in the existing code and wasn't covered by the existing tests.